packages/path_provider/README.md

```diff
@@ -8,8 +8,8 @@ This package is not an _endorsed_ implementation of `path_provider`. Therefore,
 
 ```yaml
 dependencies:
-  path_provider: ^2.0.1
-  path_provider_tizen: ^2.0.0
+  path_provider: ^2.0.2
+  path_provider_tizen: ^2.0.1
```

Comment:
This patch only fixes test-related code. nevertheless, do we have to up version?

Reply:
As far as I understand, a version specifies a fixed state of every content of a package. I remember reading it somewhere in one of the dart or flutter docs, I'll try to link the doc if I find it.

But if you think about it, from a test runner's perspective, the code has changed how it runs and therefore the output will be different to the runner. It may not be worth releasing it, but we haven't used build number versioning in any of our packages so I didn't want to create an exception.